s64da_benchmark_toolkit/streams.py

```diff
@@ -101,7 +101,11 @@ def run(self):
     def get_stream_sequence(self, stream_id):
         streams_path = os.path.join(self.benchmark.base_dir, 'queries', 'streams.yaml')
         with open(streams_path, 'r') as streams_file:
-            return yaml.load(streams_file, Loader=yaml.Loader)[stream_id]
+            streams = yaml.load(streams_file, Loader=yaml.Loader)
+            print(streams)
+            if stream_id not in streams:
+                raise Exception('Stream file {} does not contain stream id {}'.format(streams_path, stream_id))
```

Comment:
Please use a `ValueError`, and maybe build a f-string.

Reply:
Done